### DISCUSSION TITLE
Gnome overview style taskview

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#2693] [Ideas]
Consider adding the following functionality:

 * In the task view, show the taskbar even if it's hidden.
 *  Windows key opens the taskview
 *  Implement type-to-search in the taskview, i.e. any alphanumeric keystroke gets sent to the window start/search menu.

This would essentially reproduce gnome overview functionality, enabling the user to have a completely empty main workspace, maximizing space for open windows and eliminating distractions.

Here's Gnome Overview for reference. Of course the proposed hack to the taskview would just be the regular taskview with a Windows panel but the functionality would essentially be the same.

![image](https://github.com/valinet/ExplorerPatcher/assets/43324854/f65ba29e-09ed-4c6c-9107-776b489f5972)

**Comment by pyrates999** [comment]
Pressing the windows key itself has opened the start menu since the start menu was implemented in windows 95.  I doubt that will be changed.

I think the other requests for this are beyond what EP is meant for.

**Comment by tim-gromeyer** [comment]
That would be really nice. I only use Windows for work, but I still do most of my work in a Linux VM running Ubuntu (which uses Gnome). Having a Gnome-like overview in Windows would be a pleasure.
